pom.xml

```diff
@@ -169,6 +169,7 @@
         <ldapsdk.version>6.0.5</ldapsdk.version>
         <log4j.version>2.19.0</log4j.version>
         <mysql.jdbc.version>8.0.27</mysql.jdbc.version>
+        <phoenix.version>5.0.0-HBase-2.0</phoenix.version>
```

Comment:
why choose this version?

Reply:
> why choose this version?

 use latest version https://mvnrepository.com/artifact/org.apache.phoenix/phoenix-queryserver-client/6.0.0 ？

Reply:
I'm not familiar w/ phoenix ecosystem, the decision should be based on the client/server compatible matrix, and adoption rate of each project versions

Reply:
> I'm not familiar w/ phoenix ecosystem, the decision should be based on the client/server compatible matrix, and adoption rate of each project versions
thanks your advice，Phoenix thin JDBC driver is backward compatible. Phoenix thin JDBC driver 6.xx is for integration test. When using kyuubi phoenix jdbc engine, users need to provide their own jdbc driver jar, copy to 'apache kyuubi xxx embedding bin/externals/engines/jdbc' directory, which is similar to kyuubi doris jdbc engine